cmd/plasmacli/store/keystore.go

```diff
@@ -214,6 +217,76 @@ func ImportECDSA(name string, pk *ecdsa.PrivateKey) (ethcmn.Address, error) {
 
 }
 
+func Import(name string, keyJSON []byte) (ethcmn.Address, error) {
+	dir := getDir(accountsDir)
+	db, err := leveldb.OpenFile(dir, nil)
+	if err != nil {
+		return ethcmn.Address{}, fmt.Errorf("leveldb: %s", err)
+	}
+	defer db.Close()
+
+	buf := cosmoscli.BufferStdin()
+	pass, err := cosmoscli.GetPassword(ImportPassphrasePrompt, buf)
+	if err != nil {
+		return ethcmn.Address{}, fmt.Errorf("error with imported password: %s", err)
```

Comment:
just `error with password` would make sense. The message could imply there's something wrong with the file and not just trying to read from stdin